cpp/src/arrow/util/decimal_test.cc

```diff
@@ -113,7 +113,7 @@ TEST(DecimalTest, TestStringRoundTrip) {
       (1ull << 63),
       std::numeric_limits<uint64_t>::max(),
   };
-  static constexpr int32_t kScales[] = {-10, -1, 0, 1, 10};
+  static constexpr int32_t kScales[] = {0, 1, 10};
```

Comment:
would it make sense to add another test how this code behaves when scale is negative or is that already tested elsewhere?

Reply:
It's already tested at https://github.com/apache/arrow/blob/master/cpp/src/arrow/util/decimal_test.cc#L423-L433.